lib/tasks/foodsoft.rake

```diff
@@ -63,6 +63,19 @@ namespace :foodsoft do
     server.start
     server.join
   end
+
+  desc "Import and assign bank transactions"
+  task :import_and_assign_bank_transactions => :environment do
+    if FoodsoftConfig[:use_bank_accounts]
+      for ba in BankAccount.all
+        import_method = ba.find_import_method
+        next unless import_method
+        import_count = import_method.call(ba)
+        assign_count = ba.assign_unchecked_transactions
+        rake_say "#{ba.name}: imported #{import_count}, assigned #{assign_count}"
+      end
+    end
+  end
```

Comment:
I wonder if there would be a way to emit a notice that `use_bank_accounts` is not enabled when running this task manually.

Reply:
for what? if there is no UI for seeing the transactions nobody would expect that something gets imported...

Reply:
from a logging/debugging point of view, it might be useful.   i'm not sure if `rake_say` or `Rails.logger` is the best way.

`rake_say "bank accounts are not enabled so skipping import_and_assign_bank_transactions"`